src/main/java/emu/grasscutter/game/props/ItemUseAction/ItemUseUnlockHomeModule.java

```diff
@@ -14,6 +14,13 @@ public ItemUseUnlockHomeModule(String[] useParam) {
 
     @Override
     public boolean useItem(UseItemParams params) {
-        return false;
+        return true;
+    }
+
+    @Override
+    public boolean postUseItem(UseItemParams params){
+
```

Comment:
remove whitespace

Reply:
Whitespace and formatting should be fixed now.